sources/packages/web/src/helpers/RouteHelper.ts

```diff
@@ -1,18 +1,103 @@
-import { AppRoutes } from "../types";
-import { ClientIdType } from "../types/contracts/ConfigContract";
-
-export class RouteHelper {
-  static getRootRoute(clientType: ClientIdType): AppRoutes {
-    switch (clientType) {
-      case ClientIdType.STUDENT:
-        return AppRoutes.StudentRoot;
-      case ClientIdType.INSTITUTION:
-        return AppRoutes.InstitutionRoot;
-    }
-  }
-
-  static isRootRoute(path: string, clientType: ClientIdType) {
-    const root = RouteHelper.getRootRoute(clientType);
-    return path === root;
-  }
-}
+import { AuthService } from "@/services/AuthService";
+import { AppIDPType, AppRoutes, AuthStatus } from "../types";
+import { ClientIdType } from "../types/contracts/ConfigContract";
+
+export class RouteHelper {
+  static getAbsoluteRootRoute(clientType: ClientIdType): string {

```

Comment:
Comments missing for this function, getRootRoute and isRootRoute

Reply:
We do not have a team agreement to comment on every single method but I do not mind starting it.
Usually, I have been adding comments when the method is more complex or requires some business explanation.
If we start, it must be followed by everyone and every PR.
Does it make sense @ann-aot @guru-aot?

Reply:
Comments added 👍